subcommands/devices/list.go

```diff
@@ -183,10 +183,13 @@ func showDeviceList(dl *client.DeviceList, showColumns []string) {
 			if len(arg) > 2 && arg[:2] == "-p" {
 				fmt.Printf("-p%d ", showPage+1)
 				found = true
-			} else if len(arg) == 6 && arg[:6] == "--page" {
+			} else if arg == "-p" || arg == "--page" {
 				fmt.Printf("-p%d ", showPage+1)
 				found = true
 				i++
+			} else if len(arg) > 7 && arg[:7] == "--page=" {
+				fmt.Printf("-p%d ", showPage+1)
+				found = true
```

Comment:
I'm sorry for nitpicking, but this can also be combined with the above as:
`if (len(arg) > 2 && arg[:2] == "-p") || (len(arg) > 7 && arg[:7] == "--page=") {...}`

Reply:
not a nit. it cleans things up. force push fixed